dl/src/main/scala/com/intel/analytics/bigdl/nn/Cosine.scala

```diff
@@ -182,6 +183,10 @@ class Cosine[T: ClassTag](val inputSize : Int, val outputSize : Int)(
     (Array(this.weight), Array(this.gradWeight))
   }
 
+  override def getParametersTable(): Table = {
+    T(getName() -> T("weight" -> weight, "gradWeight" -> gradWeight))
```

Comment:
Or if there are no bias and no weight then we return None. Eliminate duplicate code if possible.